target/scripts/helper-functions.sh

```diff
@@ -189,7 +189,7 @@ export -f _extract_certs_from_acme
 
 # Remove the `*.` prefix if it exists, else returns the input value
 function _strip_wildcard_prefix {
-  [[ "${1}" == "*."* ]] && echo "${1:2}" || echo "${1}"
+  [[ ${1} == "*."* ]] && echo "${1:2}" || echo "${1}"
```

Comment:
Would this also make sense to do too? (_although quite a few others in this diff could do the same, perhaps just leave as double quotes instead of singles_)

```suggestion
  [[ ${1} == '*.'* ]] && echo "${1:2}" || echo "${1}"
```

Reply:
This PR was just a quick shot yesterday. I did a regex search & replace, but only considered (if present) the left side of the equal signs (that's were quotes are not necessary).
Personally, I would stick with the double quotes on the right side, because this way also `$vars` can be used inside the  quotes.